src/data-mgt/kafka/connectors/kcca-source-connector/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+project.version=1.0.0
```

Comment:
Could we relocate this kafka code to the `java` subdirectory of the `data-mgt` micro?